webrtc.html

```diff
@@ -3624,6 +3664,106 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
       </section>
 
       <section>
+        <!-- TODO: I'm sure we can think of a more creative name. -->
+        <h3>RTCRtpTransceiver Interface</h3>
+
+        <p>The <a><code>RTCRtpTransceiver</code></a> interface
+        represents a combination of an <code>RTCRtpSender</code> and
+        an <a><code>RTCRtpReceiver</code></a> that share a
+        common<code>mid</code>.
+
+        <dl class="idl" title="dictionary RTCRtpTransceiverInit">
+          <dt>boolean send = true</dt>
+
+          <dd>
+            <p>If true, indicates that
+            the <a><code>RTCRtpTransceiver</code></a>'s <a><code>RTCRtpSender</code></a>
+            will offer to send RTP and send RTP if the remote peer
+            accepts.  If false, indicates that
+            the <a><code>RTCRtpSender</code></a> will not offer to
+            send RTP and will not send RTP.</p>
+          </dd>
+
+          <dt>boolean receive = true</dt>
+
+          <dd>
+            <p>If true, indicates that
+            the <a><code>RTCRtpTransceiver</code>'</a>s <a><code>RTCRtpReceiver</code></a>
+            will offer to receive RTP and receive RTP if the remote
+            peer accepts.  If false, indicates that
+            the <a><code>RTCRtpReceiver</code></a> will not offer to
+            receive RTP and will not receive RTP.</p>
```

Comment:
Since we talk about "sendonly" and "sendrecv" elsewhere in the document, do we want to say that this causes the m-line to be "sendonly" or "Inactive"?